gnosis/safe-react
# Feature 309: Listen for web3connect disconnect event

This PR:
- Implements #309 
- For some reason, the 'disconnect' event is triggered twice, so I set `preventDuplicate: true` for wallet disconnected notification

To test:
1. Connect gnosis safe rinkeby wallet with walletconnect
2. Close walletconnect session in the app

You should see interface reset and "Wallet Disconnected" notification
